### DISCUSSION TITLE
v0.77.0

### DISCUSSION THREAD

**Discussion by tommoor** [outline/outline#7032] [Announcements]
## What's Changed

This release changes how documents are rendered, allowing the introduction of new editor functionality and attributes. Highlighted for this release is a large set of improvements to tables:

![image](https://github.com/outline/outline/assets/380914/6109e4a5-fc5a-4a25-ae33-eebbe9adc0db)

- **Column resizing:** You can now resize columns by dragging the border between them
- **Full-width tables:** Tables now expand to the full width of the editor
- **More content:** Table cells now support embeds, code blocks, and all other content types
- **Toggleable headers:** You can now toggle a header row and column
- **Improved overflow:** Tables now scroll horizontally when they're too wide for the editor
- **Improved row and column controls**: We've made it one click to add rows and columns

### Other improvements
* Added more highlighter color choices
* New collection permissions UI
* Big rendering performance improvements for documents with many images
* Add `Manage` permission to documents allows other users to manage sharing
* Added a permission selector
* Added support for embedded youtube clips by @Pranav2612000
* Matomo integration
* `/` block insert shortcut now works anywhere on a line or table
* Added the ability to remove a previously uploaded team logo or profile picture
* Added a menu option to create template directly

### Fixes
* Various fixes to improve the fluidity of app loading
* Image placeholders are now correctly sized when uploading retina images
* Images and PDF's are now served with inline content disposition
* Added support for Figma's new link formats and Figjam boards by @RenderCoder
* `files.get` now supports range header, fixes video loading
* Fixed an issue where zoomed images could get stuck
* Share button was removed on templates
* Fixed race condition that could result
* Removed trust of `state.host`
* Collapsed sidebar now works better

## New Contributors
* @lng2020 made their first contribution
* @RenderCoder made their first contribution
* @Frzk made their first contribution
* @bkrausz made their first contribution

**Full Changelog**: https://github.com/outline/outline/compare/v0.76.0...v0.77.0

**Comment by dantesia** [comment]
`Added an embedded Yandex Maps component by @shashinma
`
is this definitely included in the update?
nothing related to yandex maps is visible in the changes...

Thanks!

**Reply by tommoor** [reply]
This is extremely weird, it was PR'd and merged – but now the PR shows as empty, so it looks like something isn't quite right here and the functionality did not make it

https://github.com/outline/outline/pull/6881

**Reply by dantesia** [reply]
> This is extremely weird, it was PR'd and merged – but now the PR shows as empty, so it looks like something isn't quite right here and the functionality did not make it
> 
> #6881

i understand, thx

wait for the next release? or is it possible to add something for testing now? or can you release a fix v0.77.2? 
or just make another commit?)
again thx!

**Reply by tommoor** [reply]
This wasn't my work and now there is no record of what it looked like, which shouldn't be possible in git but…
